printNamesExpress.js

```diff
@@ -0,0 +1,17 @@
+//program to print with express
+const fs = require("fs");
+const express = require("express");
+const app = express();
+app.listen(3615);
+const myfunc = (re1,re2)=>{
```

Comment:
I don't know what re1, re2 are. If someone new was reading these two, they would have no idea what they are. Just use the req, res. It makes it easier to understand.